x/lockup/keeper/msg_server_test.go

```diff
@@ -47,14 +47,16 @@ func (suite *KeeperTestSuite) TestMsgLockTokens() {
 	for _, test := range tests {
 		suite.SetupTest()
 
-		if test.expectPass {
-			// Setup the Account
-			err := simapp.FundAccount(suite.app.BankKeeper, suite.ctx, test.param.lockOwner, test.param.coinsInOwnerAddress)
-			suite.Require().NoError(err)
+		// lock coins
+		addr1 := sdk.AccAddress([]byte("addr1---------------"))
+		coins := sdk.Coins{sdk.NewInt64Coin("stake", 10)}
 
-			_, err = suite.app.LockupKeeper.LockTokens(suite.ctx, test.param.lockOwner, test.param.coinsToLock, test.param.duration)
-			suite.NoError(err, "test %v", test.name)
+		err := simapp.FundAccount(suite.app.BankKeeper, suite.ctx, addr1, coins)
+		suite.Require().NoError(err)
+		_, err = suite.app.LockupKeeper.LockTokens(suite.ctx, addr1, coins, time.Second)
+		suite.Require().NoError(err)
 
```

Comment:
@ValarDragon updated as suggested. 

If the approach is correct, I was thinking about moving this functionality to `x/lockup/keeper/keeper_test.go` very similar to [this](https://github.com/osmosis-labs/osmosis/blob/f2a60d546887aca5890640a11c37518e6e6a7c81/x/gamm/keeper/keeper_test.go#L80) `swap_test` setup function feature. lmk what you think